typing/printtyped.ml

```diff
@@ -330,11 +330,17 @@ and expression i ppf x =
                       expression_extra i ppf extra attrs; i+1)
       (i+1) x.exp_extra
   in
-  (match Btype.Value_mode.check_const x.exp_mode with
-  | Some Global -> line i ppf "value_mode global\n"
-  | Some Regional -> line i ppf "value_mode regional\n"
-  | Some Local -> line i ppf "value_mode local\n"
-  | None -> line i ppf "value_mode <modevar>\n");
+  ( let consts = Btype.Value_mode.check_const x.exp_mode in
+    line i ppf "value_mode ";
+    line 0 ppf (match fst consts with
```

Comment:
This is somewhat ugly as I couldn't find a way to combine two format strings (the way '^' acts on normal strings). Is there a nice operator for that?